rpc/RPCTransaction.java

```diff
@@ -110,6 +113,11 @@ public ConceptManager concepts() {
         return conceptManager;
     }
 
+    @Override
+    public LogicManager logics() {
```

Comment:
We've agreed to rename this to `logic`.
```suggestion
    public LogicManager logic() {
```